pydicom/dataset.py

```diff
@@ -344,11 +344,9 @@ class Dataset(dict):
     indent_chars = "   "
 
     # TODO: this the correct list?
-    _BINARY_VR_VALUES = ['OW', 'OB', 'OD', 'OF', 'OL', 'UN',
-                         'OW/OB', 'OW or OB', 'OB or OW', 'US or SS']
-
-    _VRs_TO_BE_FLOATS = ["DS", "FL", "FD", ]
-    _VRs_TO_BE_INTS = ["IS", "SL", "SS", "UL", "US", ]
+    _BINARY_VR_VALUES = ['OW', 'OB', 'OD', 'OF', 'OL', 'UN', 'OB or OW']
+    _VRs_TO_BE_FLOATS = ['DS', 'FL', 'FD', ]
+    _VRs_TO_BE_INTS = ['IS', 'SL', 'SS', 'UL', 'US', 'US or SS']
```

Comment:
Had a look to see if these were complete:
```
>>> from pydicom._dicom_dict import DicomDictionary
>>> set([(x[0], x[3]) for x in DicomDictionary.values() if ' or ' in x[0]])
{('US or SS', ''), ('US or OW', ''), ('US or SS or OW', 'Retired'), ('US or SS',
 'Retired'), ('OB or OW', '')}
```
So technically, there is `US or OW` and `US or SS or OW` not accounted for that could come up in both retired and non-retired data elements.  I haven't tried to follow through the logic to see what would happen if one of those came up.  Seems unlikely enough that maybe we could add another issue to look into it / add a tests for it, make sure it fails gracefully at least.

Reply:
Yes, I have seen these too, but wasn't sure if they ever will come up and how to handle them - probably handle them as binary, I guess. I can add them to the binary values, if you agree.
I also haven't added these to the tests yet, though we wouldn't gain much from it if we handle them as binary, I think.

Reply:
I think adding them is reasonable, and agree that they don't need to be added separately to the tests.